app/events/email-incoming/index.ts

```diff
@@ -32,6 +32,7 @@ interface UserData {
   sub?: string
   name?: string
   affiliation?: string
+  receiveOnly?: boolean
```

Comment:
How about two flags, `submit` and `receive`? Users can be submit only, receive only, or submit and receive. Is that right, @jracusin?

Reply:
Sure! I can add that now